app/models/word.rb

```diff
@@ -1,7 +1,45 @@
 class Word
   include Mongoid::Document
-  field :word_name, type: String
-  field :description, type: String
-  field :syntactical_category, type: String
-  field :semantical-categories, type: String
+
+  has_many :categories
+
+  field :name_de, type: String
+  field :name_en, type: String
+  field :description_de, type: String
+  field :description_en, type: String
+  field :syntactical_category_de, type: String
```

Comment:
Same here: Why do we need those categories for each language? The semantical and syntactical categories should be the same in both languages.